### PR TITLE
fix: fix data commitment ranges overlap

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Fixes https://github.com/celestiaorg/celestia-app/issues/1204

Now, the ranges will be as follows, with a data commitment window of 400:
- 0-399
- 400-799
- 800-1199
- ...
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: x/qgb/keeper/keeper_data_commitment.go [hunk 1]
```diff
@@ -7,11 +7,11 @@ import (
 
 // TODO add unit tests for all the keepers
 
-// GetCurrentDataCommitment creates latest data commitment at current height according to
+// GetCurrentDataCommitment creates the latest data commitment at current height according to
 // the data commitment window specified
 func (k Keeper) GetCurrentDataCommitment(ctx sdk.Context) (types.DataCommitment, error) {
 	beginBlock := uint64(ctx.BlockHeight()) - k.GetDataCommitmentWindowParam(ctx)
-	endBlock := uint64(ctx.BlockHeight())
+	endBlock := uint64(ctx.BlockHeight()) - 1
```
Comment (evan-forbes) [inline x/qgb/keeper/keeper_data_commitment.go:14]: [optional nit]
perhaps add a line of reasoning here

**Review by evan-forbes — DISMISSED** [review]

**Comment by evan-forbes** [comment]
also, do we need the `qgb-integration` PR as well as this one? or can we just use this one?

**Comment by SweeXordious** [comment]
> also, do we need the qgb-integration PR as well as this one? or can we just use this one?

After today's sync, we can close the other PR, will do so shortly

File: x/qgb/keeper/keeper_data_commitment.go (hunk 1 above)
Comment (SweeXordious) [reply x/qgb/keeper/keeper_data_commitment.go]: [f8ebd4b](https://github.com/celestiaorg/celestia-app/pull/1217/commits/f8ebd4b1772682c3233982fb4b5b9160f2c6b046) is this good?

**Review by evan-forbes — APPROVED** [review]
:+1: :+1:

**Pull request merged**